creator-node/src/routes/healthCheck.js

```diff
@@ -18,32 +20,13 @@ module.exports = function (app) {
       return errorResponseServerError()
     }
 
-    const ipfs = req.app.get('ipfsAPI')
-    try {
-      const start = Date.now()
-      const timestamp = start.toString()
-      const content = Buffer.from(timestamp)
-
-      // Add new buffer created from timestamp (without pin)
-      const results = await ipfs.add(content, { pin: false })
-      const hash = results[0].hash // "Qm...WW"
-
-      // Retrieve and validate hash from local node
-      const ipfsResp = await ipfs.get(hash)
-      const ipfsRespStr = ipfsResp[0].content.toString()
-      const isValidResponse = (ipfsRespStr === timestamp)
-
-      // Test pin ops if requested
-      if (req.query.pin === 'true') {
-        await ipfs.pin.add(hash)
-        await ipfs.pin.rm(hash)
-      }
-
-      const duration = `${Date.now() - start}ms`
-      return successResponse({ hash, isValidResponse, duration })
-    } catch (e) {
-      return errorResponseServerError({ error: e })
+    const [value] = await getMonitors([MONITORS.IPFS_READ_WRITE_STATUS])
+    if (!value) {
+      return errorResponseServerError({ error: 'IPFS not healthy' })
     }
+
+    const { hash, duration } = value
```

Comment:
this changes return object by removing `isValidResponse` field - looks fine to me but just confirming you did this intentionally and checked?

Reply:
yeah, i checked, we weren't using this anywhere

Reply:
💯